src/dotnet/devcontainer-feature.json

```diff
@@ -1,30 +1,30 @@
 {
     "id": "dotnet",
-    "version": "1.1.3",
-    "name": "Dotnet CLI",
+    "version": "2.0.0",
+    "name": ".NET",
     "documentationURL": "https://github.com/devcontainers/features/tree/main/src/dotnet",
-    "description": "Installs the .NET CLI. Provides option of installing sdk or runtime, and option of versions to install. Uses latest version of .NET sdk as defaults to install.",
+    "description": "Installs .NET and its CLI. Offers options to install a specific version of .NET, additional versions of .NET, or just the .NET runtime.",
     "options": {
         "version": {
             "type": "string",
             "proposals": [
                 "latest",
                 "7",
                 "6",
-                "3.1"
+                "3"
             ],
             "default": "latest",
-            "description": "Select or enter a dotnet CLI version. (Available versions may vary by Linux distribution.)"
+            "description": "Version of .NET to install. Use 'latest' to install the latest LTS version."
```

Comment:
This supports `x.y.z` versions too right?  I know it didn't say this before, but might be nice to point this out in the description here.

Reply:
Good idea, I updated the descriptions